pytest.ini

```diff
@@ -10,3 +10,4 @@ markers =
     xdist_incompatible: test which compiles a file also compiled by another test
     external: test using an external dll (problematic with conda on Windows)
     cuda: test to generate cuda code
+    gpu: test related to gpu
```

Comment:
Where do you use this new mark?

Reply:
u right there is not use for that for the current time fixed https://github.com/pyccel/pyccel-cuda/commit/5f2df095d7f5a044cd18eef9d1777565897847d3